blockbase/inc/social-navigation.php

```diff
@@ -13,11 +13,6 @@ function blockbase_condition_to_render_social_menu( $block_content, $block ) {
 		return false;
 	}
 
-	// The block should have an unstable location attribute.
```

Comment:
Is there still value in including this now that the __unstableLocation is gone?  
I don't see any themes that provide a `custom.navigation/social-links` value.

What other value does this provide?  Can we get rid of it completely?

Reply:
> Can we get rid of it completely?

I think so, but is there potential for it to break how social menus are displayed for some sites running Blockbase or its children? There are five header patterns that would appear to use this filter:

https://github.com/Automattic/themes/blob/trunk/blockbase/parts/header-default.html#L19
https://github.com/Automattic/themes/blob/trunk/blockbase/parts/header-linear.html#L19
https://github.com/Automattic/themes/blob/trunk/blockbase/parts/header-minimal.html#L21
https://github.com/Automattic/themes/blob/trunk/blockbase/parts/header-rounded-logo.html#L19
https://github.com/Automattic/themes/blob/trunk/blockbase/parts/header-wide.html#L19

Reply:
Yes, I agree that for that reason it should stay.  The remainder of what's left is unconnected to `__unstableLocation` and removing it just removes functionality.